remote-storage-managers/hdfs/src/main/java/org/apache/kafka/rsm/hdfs/HDFSRemoteStorageManager.java

```diff
@@ -370,16 +423,14 @@ public void close() {
      */
     @Override
     public void configure(Map<String, ?> configs) {
-        fsURI = URI.create(configs.get(HDFS_URI_PROP).toString());
-        baseDir = configs.get(HDFS_BASE_DIR_PROP).toString();
-
-        String intervalBytesStr = null;
-        if (configs.containsKey(HDFS_REMOTE_INDEX_INTERVAL_BYTES))
-            intervalBytesStr = configs.get(HDFS_REMOTE_INDEX_INTERVAL_BYTES).toString();
-
-        if (intervalBytesStr == null || intervalBytesStr.isEmpty())
-            intervalBytesStr = HDFS_REMOTE_INDEX_INTERVAL_BYTES_DEFAULT;
-        indexIntervalBytes = Integer.parseInt(intervalBytesStr);
+        HDFSRemoteStorageManagerConfig conf = new HDFSRemoteStorageManagerConfig(configs, true);
+
+        fsURI = URI.create(conf.getString(HDFSRemoteStorageManagerConfig.HDFS_URI_PROP));
+        baseDir = conf.getString(HDFSRemoteStorageManagerConfig.HDFS_BASE_DIR_PROP);
+        indexIntervalBytes = conf.getInt(HDFSRemoteStorageManagerConfig.HDFS_REMOTE_INDEX_INTERVAL_BYTES_PROP);
+        cacheLineSize = conf.getInt(HDFSRemoteStorageManagerConfig.HDFS_REMOTE_READ_MB_PROP) * 1048576;
+        long cacheSize = conf.getInt(HDFSRemoteStorageManagerConfig.HDFS_REMOTE_READ_CACHE_MB_PROP) * 1048576L;
```

Comment:
good to have a check that this value should be more than `cacheLineSize` and mention in the docs of those properties.